### PR TITLE
Primeira versão do dataset dplyr::starwars

### PR DESCRIPTION
**Pull request by beatrizmilz**
Base gerada: https://drive.google.com/file/d/1SpisQDQS743SftjvN6lIywKdfF3wrEyH/view?usp=sharing

Olá, essa é a primeira versão da tradução do dataset dplyr::starwars. #70
Alguém poderia revisar a tradução, por favor?

```
> dplyr::glimpse(dados_starwars)
Rows: 87
Columns: 14
$ nome            <chr> "Luke Skywalker", "C-3PO", "R2-D2", "Darth Vader", "Leia Orga…
$ altura          <int> 172, 167, 96, 202, 150, 178, 165, 97, 183, 182, 188, 180, 228…
$ massa           <dbl> 77.0, 75.0, 32.0, 136.0, 49.0, 120.0, 75.0, 32.0, 84.0, 77.0,…
$ cor_do_cabelo   <chr> "Loiro", NA, NA, "Nenhum", "Castanho", "Castanho, Cinza", "Ca…
$ cor_da_pele     <chr> "Branca clara", "Ouro", "Branca, Azul", "Branca", "Clara", "C…
$ cor_dos_olhos   <chr> "Azul", "Amarelo", "Vermelho", "Amarelo", "Castanho", "Azul",…
$ ano_nascimento  <dbl> 19.0, 112.0, 33.0, 41.9, 19.0, 52.0, 47.0, NA, 24.0, 57.0, 41…
$ sexo_biologico  <chr> "Macho", "Nenhum", "Nenhum", "Macho", "Fêmea", "Macho", "Fême…
$ genero          <chr> "Masculino", "Masculino", "Masculino", "Masculino", "Feminino…
$ planeta_natal   <chr> "Tatooine", "Tatooine", "Naboo", "Tatooine", "Alderaan", "Tat…
$ especie         <chr> "Humano", "Droide", "Droide", "Humano", "Humano", "Humano", "…
$ filmes          <list> "O Império Contra-Ataca | A Vingança dos Sith | O Retorno de…
$ veiculos        <list> <"Snowspeeder", "Imperial Speeder Bike">, <>, <>, <>, "Imper…
$ naves_espaciais <list> <"X-wing", "Imperial shuttle">, <>, <>, "TIE Advanced x1", <…
```

Alguns pontos:
- Não traduzi os elementos das colunas: nome, planeta natal, veiculos, naves espaciais.
- Traduzi apenas 3 elementos da coluna especie: Humano, Droide, Espécie do Yoda. Os outros estão na forma original.
- A tradução das cores de pele foram um pouco confusas pra mim. Seria bom a  pessoa revisora comentar se discorda de algo.

- A coluna filmes é uma lista. Ao traduzir o nome de cada filme, os elementos que apresentavam mais de um filme não foram traduzidos. Então traduzi todas as combinações... Infelizmente não entendi o que precisamos fazer para manter a estrutura igual à original.
```
> dados_starwars$filmes[1:3]
[[1]]
[1] "O Império Contra-Ataca | A Vingança dos Sith | O Retorno de Jedi | Uma Nova Esperança | O Despertar da Força"

[[2]]
[1] "O Império Contra-Ataca | Ataque dos Clones | A Ameaça Fantasma | A Vingança dos Sith | O Retorno de Jedi | Uma Nova Esperança"

[[3]]
[1] "O Império Contra-Ataca | Ataque dos Clones | A Ameaça Fantasma | A Vingança dos Sith | O Retorno de Jedi | Uma Nova Esperança | O Despertar da Força"

> dplyr::starwars$films[1:3]
[[1]]
[1] "The Empire Strikes Back" "Revenge of the Sith"     "Return of the Jedi"     
[4] "A New Hope"              "The Force Awakens"      

[[2]]
[1] "The Empire Strikes Back" "Attack of the Clones"    "The Phantom Menace"     
[4] "Revenge of the Sith"     "Return of the Jedi"      "A New Hope"             

[[3]]
[1] "The Empire Strikes Back" "Attack of the Clones"    "The Phantom Menace"     
[4] "Revenge of the Sith"     "Return of the Jedi"      "A New Hope"             
[7] "The Force Awakens"  
```

### CODE REVIEW

File: inst/specs/starwars.yml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    desc: Cor dos olhos
+    values:
+      blue: Azul
+      yellow: Amarelo
+      red: Vermelho
+      brown: Castanho
+      blue-gray: Azul acinzentado
+      black: Preto
+      orange: Laranja
+      hazel: Castanho claro
```
Comment (beatrizmilz) [inline inst/specs/starwars.yml:76]:
```suggestion
      hazel: Castanho-esverdeado
```

File: inst/specs/starwars.yml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      blonde: Loiro
+      none: Nenhum
+      brown: Castanho
+      brown, grey: Castanho, Cinza
+      grey: Cinza
+      black: Preto
+      auburn, white: Ruivo, Branco
+      auburn, grey: Ruivo, Cinza
+      white: Branco
+      auburn: Ruivo
```
Comment (beatrizmilz) [inline inst/specs/starwars.yml:28]:
```suggestion
      auburn: Castanho-avermelhado
```

**Review by beatrizmilz — COMMENTED** [review]
Sugestões da @EmmanuelleRN

File: inst/specs/starwars.yml [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+      blonde: Loiro
+      none: Nenhum
+      brown: Castanho
+      brown, grey: Castanho, Cinza
+      grey: Cinza
+      black: Preto
+      auburn, white: Ruivo, Branco
+      auburn, grey: Ruivo, Cinza
+      white: Branco
+      auburn: Castanho-avermelhado
```
Comment (beatrizmilz) [inline inst/specs/starwars.yml:28]:
```suggestion
      auburn: Castanho avermelhado
```

File: inst/specs/starwars.yml [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    desc: Cor dos olhos
+    values:
+      blue: Azul
+      yellow: Amarelo
+      red: Vermelho
+      brown: Castanho
+      blue-gray: Azul acinzentado
+      black: Preto
+      orange: Laranja
+      hazel: Castanho-esverdeado
```
Comment (beatrizmilz) [inline inst/specs/starwars.yml:76]:
```suggestion
      hazel: Castanho esverdeado
```

**Review by beatrizmilz — COMMENTED** [review]
Percebi que estava no padrão diferente, removi os -

File: inst/specs/starwars.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (beatrizmilz) [reply inst/specs/starwars.yml]: Obrigada! Alterei isso :D

File: inst/specs/starwars.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (beatrizmilz) [reply inst/specs/starwars.yml]: Obrigada! Alterei isso :D

**Comment by beatrizmilz** [comment]
@rivaquiroga in this dataset, we have some columns that are lists. I don't know if I translated in the best way. Do you remember any case in datos that had the same situation, so I can look at the code to see how it was translated?  

Example:

``` r
library(magrittr)
dplyr::starwars %>% 
  dplyr::select(films:starships)
#> # A tibble: 87 x 3
#>    films     vehicles  starships
#>    <list>    <list>    <list>   
#>  1 <chr [5]> <chr [2]> <chr [2]>
#>  2 <chr [6]> <chr [0]> <chr [0]>
#>  3 <chr [7]> <chr [0]> <chr [0]>
#>  4 <chr [4]> <chr [0]> <chr [1]>
#>  5 <chr [5]> <chr [1]> <chr [0]>
#>  6 <chr [3]> <chr [0]> <chr [0]>
#>  7 <chr [3]> <chr [0]> <chr [0]>
#>  8 <chr [1]> <chr [0]> <chr [0]>
#>  9 <chr [1]> <chr [0]> <chr [1]>
#> 10 <chr [6]> <chr [1]> <chr [5]>
#> # … with 77 more rows
```

<sup>Created on 2021-04-08 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

**Comment by rivaquiroga** [comment]
Hi, @beatrizmilz! We don't have datasets with list-columns in {datos}. I'm not sure if the translation works for those cases. I'll check and get back to you in a couple of days.

**Comment by rivaquiroga** [comment]
@beatrizmilz, for list-columns, the YAML format should be:
`<"Attack of the Clones", "Revenge of the Sith">: ["Ataque dos Clones", "A Vingança dos Sith"]`

I just fixed it:

```r
> dados_starwars %>% 
+   dplyr::select(filmes:naves_espaciais) 
# A tibble: 87 x 3
   filmes    veiculos  naves_espaciais
   <list>    <list>    <list>         
 1 <chr [5]> <chr [2]> <chr [2]>      
 2 <chr [6]> <chr [0]> <chr [0]>      
 3 <chr [7]> <chr [0]> <chr [0]>      
 4 <chr [4]> <chr [0]> <chr [1]>      
 5 <chr [5]> <chr [1]> <chr [0]>      
 6 <chr [3]> <chr [0]> <chr [0]>      
 7 <chr [3]> <chr [0]> <chr [0]>      
 8 <chr [1]> <chr [0]> <chr [0]>      
 9 <chr [1]> <chr [0]> <chr [1]>      
10 <chr [6]> <chr [1]> <chr [5]>      
# … with 77 more rows
```

**Comment by rivaquiroga** [comment]
@beatrizmilz, I merged the last changes from the main branch to this one + updated the website.  Unless there are more changes to make to this dataset, this PR is ready to be merge.

**Comment by beatrizmilz** [comment]
Hi Riva! Thank you for working on this.
I tested the code in this branch but it seems that it's not all translated:

```
devtools::load_all()
dados_starwars <- translate("starwars.yml")
```
This is how is appearing in the View: the lines that only have 1 movie got translated. The others are still in English...
image

I`ll try to understand more to see if I can find a solution.

**Comment by rivaquiroga** [comment]
I'll take a look during the afternoon!

**Comment by rivaquiroga** [comment]
I updated the branch to the intermediate solution: making the "filmes" column a list. 
We still have to check if it is possible with the current translation method not to merge everything into one character string.

**Comment by beatrizmilz** [comment]
Thank you Riva! Do you think is ok to merge as it is, and we can after check if that is possible?

**Comment by rivaquiroga** [comment]
Sure! Although, maybe it would be better to left the titles untranslated for the moment, so people can use the dataset in a similar way as the original. What do you think?

**Comment by beatrizmilz** [comment]
Hi Riva! Good idea. I comment the whole section of `values` for dados_starwars$filmes . So now all the names of the movies are in English!

**Review by rivaquiroga — APPROVED** [review]

**Comment by beatrizmilz** [comment]
Thanks Riva!

**Pull request merged**